### ISSUE TITLE
Black Screen during fast forward - Ubuntu 14.04.4 - Openshot  2.0.6

### ISSUE THREAD

**Issue by cnxsoft** [OpenShot/openshot-qt#109]
I've changed the timescale to some part of a video to 8x fast forward. Then select export video to h.264 (AVI) 1080p30 (the project shows HDV_720_24p, not sure that matters). The process will crash sometimes, but when it works, the accelerated part of the video will just show a black zone for the duration of the "fast forward" zone before resuming playback. I've also noted in the remaining of the video the image is shifted, left&upwards so I have have large black bands on the bottom and right side of the video.

**Comment by cnxsoft** [comment]
That's the project file if it is useful: https://dl.dropboxusercontent.com/u/45842273/k1_plus_dvb_setup.osp

I've got two zones with fast forward, and the problem appears to be random, as I've now generated a video where the first fast forward zone is fine, but the second is blacked out. The resulting video has been uploaded to YouTube https://www.youtube.com/watch?v=RJ39UHGvrgs.

The issue starts at around 3:38 on the video.
